src/assets/scripts/client/aircraft/StripView/StripViewController.js

```diff
@@ -156,7 +153,10 @@ export default class StripViewController {
         const stripViewModel = new StripViewModel(aircraftModel, stripViewCid);
 
         this._collection.addItem(stripViewModel);
-        this._addViewToStripList(stripViewModel);
+
+        if (aircraftModel.isDeparture() || aircraftModel.inside_ctr) {
+            this._addViewToStripList(stripViewModel);
```

Comment:
Could we get a description on the docblock for `#collection` to differentiate it from wherever we're "adding" this strip view model (apparently to the `.$stripViewList`)?

Reply:
yeah, one is a collection the other is a DOM list.  updated.